src/styles/blocks/page.scss

```diff
@@ -0,0 +1,45 @@
+:root {
+  background-color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.page {
+  &__body {
+    display: grid;
+    grid-template: 60px 1fr / 1fr;
+    grid-template-areas: "header"
+      "main";
+  }
+
+  &__header {
+    display: grid;
+    align-items: center;
+    padding: 0 50px;
+    grid-area: header;
+    grid-template: minmax(60px, 1fr) / minmax(auto, 300px) minmax(auto, 1100px);
+  }
+
+  &__main {
+    display: grid;
+    grid-area: main;
+    width: 100%;
+    padding: 50px 40px 72px;
+    grid-template-columns: repeat(auto-fit,200px);
```

Comment:
To ensure there are always 4 cards in a row, change grid-template-columns to 'repeat(4, 1fr)' instead of 'repeat(auto-fit, 200px)'.